stable/nats/values.yaml

```diff
@@ -250,36 +250,45 @@ ingress:
   #   key:
   #   certificate:
 
-sidecars:
-## Add sidecars to the pod.
-## e.g.
-# - name: your-image-name
-  # image: your-image
-  # imagePullPolicy: Always
-  # ports:
-  # - name: portname
-  #   containerPort: 1234
-
-## Prometheus Exporter / Metrics
+## Metrics / Prometheus NATS Exporter
 ##
+## ref: https://github.com/nats-io/prometheus-nats-exporter
 metrics:
   enabled: false
   image:
     registry: docker.io
-    repository: appcelerator/prometheus-nats-exporter
-    tag: 0.17.0
+    repository: synadia/prometheus-nats-exporter
+    tag: 0.1.0
     pullPolicy: IfNotPresent
     ## Optionally specify an array of imagePullSecrets.
     ## Secrets must be manually created in the namespace.
     ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
     ##
     # pullSecrets:
     #   - myRegistrKeySecretName
-   ## Metrics exporter resource requests and limits
+  ## Metrics exporter resource requests and limits
   ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
   ##
   # resources: {}
-   ## Metrics exporter pod Annotation
-  podAnnotations:
+  ## Metrics exporter port
+  port: 7777
+  ## Metrics exporter annotations
+  annotations:
```

Comment:
Why did you change it from `podAnnotations` to `annotations`? Please note that this kind of changes will break backwards compatibility and you won't be able to upgrade the chart from previous version by running:

```
helm upgrade ...
```